win-linux/package/windows/update/changes/7.3.0/en.html

```diff
@@ -1,20 +1,69 @@
 <h2>New Features</h2>
 <ul>
-    <li>Changed options for app updating (Windows 7+)</li>
+    <li>Unified appearance for tooltips (Windows, Linux)</li>
+    <li>Window shadow effect (Windows only)</li>
+    <li>Jump List (Tasks) for the application shortcut in the start menu and taskbar (Windows 7 and later)</li>
+    <li>Abandoning the third-party update window and reworking the Check for updates automatically option (Windows 7 and later)</li>
+    <li>Dialog windows (file manager and print) are replaced with the system ones (Linux only)</li>
+    <li>Support for xdg-desktop-portal in the file dialog window (Linux only)</li>
+    <li>Use the file name when printing to PDF by default (Linux only)</li>
+    <li> quick access panel</li>
+    <li>3D Rotation settings for 3D charts</li>
+    <li>Display of chart error bars (for opening only)</li>
+    <li>Inserting Smart Art objects</li>
+    <li>Uzbek dictionaries for spell checking: `Uzbek (Cyrillic)` and `Uzbek (Latin)`</li>
+    <li>Presets for inserting horizontal and vertical text boxes</li>
+    <li>Ability to hide left and right panel on the `View` tab of the top toolbar</li>
+    <li>Width of the styles / themes panel now fits the whole number of items</li>
+    <li>Grouping for table templates</li>
+    <li>Cell styles in the OLE object editor</li>
+    <li>Ability to resize dialog windows for editing charts, OLE objects, and mail merge recipients</li>
+    <li>Unified appearance for dialog windows, context menus, toolbar, etc.</li>
+    <li>Optimizing display of comments</li>
+    <li>Changed metafiles conversion to SVG</li>
+    <li>Reading and writing PDF are combined in a single library to optimize work</li>
+    <li>Quick print function (for Windows and Linux)</li>
+    <li>Changed default logic of work with local documents locked for editing (for Windows and Linux)</li>
```

Comment:
<li>Changed default logic of work with local documents locked for editing (Windows, Linux)</li>